src/css/components/panel.scss

```diff
@@ -40,6 +40,18 @@ $panel-padding: $grid-3;
   flex-direction: row;
   justify-content: flex-start;
   margin: 0;
+
+  p, span {
+    font-size: 0.9rem;
+  }
+}
+
+* + .panel-actions {
+  margin-top: $grid-1;
+}
+
+.panel-actions + * {
+  margin-top: $grid-2;
```

Comment:
Yeah, I figured there would be several, but still known and you would end up with something like
```scss
.panel-actions {
  & + .panel-rows,
  & + .panel-actions,
  & + .panel-documentation {
  /* ... */
  }
}
```